### PR TITLE
Added new switch, knob, and lever sounds

### PR DESCRIPTION
**Pull request by hotshotp**
## Summary of Changes

Here we worked with tons of annoying templates and mind numbing work to bring you these sounds. As before, here we have the small changes:

- Improved the sound of the runway light thumps
- Made engine shutdown quieter in cockpit

Here we have the main, fat changes that are the best part of the PR:

- Changed the sound of the parking brake
- Changed the sound of the spoiler lever
- Changed the sound of the flap lever
- Changed the sound of the Engine master switches
- Changed the sound of the knobs such as the engine mode selector and ND Range knob
- Changed the sound of the RMP knobs
- Changed the sound of the QNH, and FCU altitude speed heading and VS speed knobs
- Changed the sound of the ADIRS knobs (separate to other knobs because IRL you would pull them before turning)
- Changed the sound of the cockpit door lock switch
- Changed the switch sound for seatbelt sign, no smoking sign, and that type of switch

Important: Button sounds and light switch sounds are coming in a separate PR - number #1877 - this will rework the buttons on the OVHD panel and the 8 light switches which will stay default for this PR

## References

These were recorded by komp in a cold and dark a320 cockpit, so everything is genuine and realistic - also confirmed by other pilots to sound realistic

## Additional context

templates really suck 

Discord username (if different from GitHub):

Boris#9134

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: A32NX/ModelBehaviorDefs/A32NX/Interior/A32NX_Interior_Autopilot.xml [hunk 1]
```diff
@@ -132,5 +132,308 @@
...
+						<ANIMTIP_1_ON_CURSOR>TurnRight</ANIMTIP_1_ON_CURSOR> -->
+						<ANIMTIP_2_ON_CURSOR>UpArrow</ANIMTIP_2_ON_CURSOR>
+						<ANIMTIP_3_ON_CURSOR>DownArrow</ANIMTIP_3_ON_CURSOR>
+						<ANIMTIP_0>TT:COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_SPEED_DECREASE</ANIMTIP_0>
+						<ANIMTIP_1>TT:COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_SPEED_INCREASE</ANIMTIP_1>
+						<ANIMTIP_2>TT:COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_SPEED_ENGAGE_MANAGE_MODE</ANIMTIP_2>
+						<ANIMTIP_3>TT:COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_SPEED_ENGAGE_SELECTED_AIRSPEED_MODE</ANIMTIP_3>
+					</UseTemplate>
+				</Case>
+				<Case Value="B747">
```
Comment (lukecologne) [inline A32NX/ModelBehaviorDefs/A32NX/Interior/A32NX_Interior_Autopilot.xml:195]: You should remove all the code that is not related to the airbus. It will make the templates much more compact and readable

Comment (hotshotp) [reply A32NX/ModelBehaviorDefs/A32NX/Interior/A32NX_Interior_Autopilot.xml:195]: Holland told me not to worry about that and that he will clean up the code later. To be honest I really don't understand templates and it's very confusing for me to edit them without messing anything else up

Comment (lukecologne) [reply A32NX/ModelBehaviorDefs/A32NX/Interior/A32NX_Interior_Autopilot.xml:195]: Ah, if he's going to fix it later that's fine.

**Comment by wpine215** [comment]
Source WWISE files?

**Comment by mikecbr** [comment]
I think , after the lastest update that the engines are too loud in cruise.... last time i flew , yesterday , (im a cabin crew), and everytime i go to cockpit , you cant hear the engines at all , even if the aircraft is still climbing  after FL10 i cant hear them at all
@hotshotp

**Comment by hotshotp** [comment]
> I think , after the lastest update that the engines are too loud in cruise.... last time i flew , yesterday , (im a cabin crew), and everytime i go to cockpit , you cant hear the engines at all , even if the aircraft is still climbing  after FL10 i cant hear them at all
> @hotshotp 

Please do not post off topic things on this PR but rather take it to discord. To answer your question, it is on your end because I did not touch anything about the wind or engines that would cause your problem
